RPi/mira-gateway.env

```diff
@@ -0,0 +1,2 @@
+MIRA_GW_PAN_ID=13243546
+MIRA_GW_KEY=11121314212223243132333441424344
```

Comment:
Should the install-script create these instead?

mira_gen_panid.py could be used to generate the pan (might need to be transformed with `printf "%08x" $(mira_gen_panid.py)`).

Hexdump can generate the key:
hexdump -n 16 -e '"%02x"' /dev/urandom

Reply:
I added generation if libmira is vendored, otherwise it uses these credentials. I didn't want to create a hard dependency on libmira, and I also didn't want to copy mira_gen_panid.py to this repo.